pkg/controller/bundle/bundle_unpacker.go

```diff
@@ -651,6 +651,14 @@ func (c *ConfigMapUnpacker) ensureJob(cmRef *corev1.ObjectReference, bundlePath
 
 		return
 	}
+	// Cleanup old unpacking job and retry
+	if _, isFailed := getCondition(job, batchv1.JobFailed); isFailed {
+		err = c.client.BatchV1().Jobs(job.GetNamespace()).Delete(context.TODO(), job.GetName(), metav1.DeleteOptions{})
+		if err == nil {
+			job, err = c.client.BatchV1().Jobs(fresh.GetNamespace()).Create(context.TODO(), fresh, metav1.CreateOptions{})
+		}
+		return
+	}
```

Comment:
This appears to retry without limit?

Reply:
What's the retry cadence? is it exp backoff?
Maybe it's ok to retry forever as long as we're not hammering the apiserver

Reply:
This runs whenever olm syncs resolves a namespace - we use the [default client-go workqueue](https://github.com/kubernetes/client-go/blob/v0.28.1/util/workqueue/default_rate_limiters.go#L39) so we have exp backoff up to ~15 min.